project/Common.scala

```diff
@@ -51,15 +57,15 @@ object Common extends AutoPlugin {
     "Use ReactiveMongo-Shaded (see system property 'reactivemongo.shaded')"
   )
 
+  lazy val scalaVersions: Seq[String] = {
+    if (actorModule == "akka") Seq(scala211, scala212, scala213, scala31)
+    else Seq(scala212, scala213)
```

Comment:
pekko supports scala3 - but only scala3.3 and above

Reply:
Common.scala31 is already set to 3.2.2. Maybe, it should be renamed to scala3 and set to 3.3.0. That would allow pekko support for ReactiveMongo support to also include scala3.

Reply:
Indeed. But at this point ReactiveMongo builds against Scala `3.2.2` https://github.com/SamTheisens/ReactiveMongo/blob/master/project/Common.scala#L34
Not sure there's anything against upgrading to `3.3.x`, but perhaps that needs to be done in a separate PR?

Reply:
3.3 reintroduce bug about compiler warnings

Reply:
@cchantep should I submit an issue for upgrading to scala 3.3.x?

Reply:
Yes, although note that Scala 3.3.1 (not yet released) is meant to fix some regressions so maybe you can test against nightly to see if its already been fixed.

Reply:
The problem has unfortunately not been fixed in the latest nightly. But I did manage to pin down which scala 3 change is responsible for the compilation error. This PR reproduces it https://github.com/ReactiveMongo/ReactiveMongo/pull/1251